carbonserver/main.py

```diff
@@ -69,4 +83,4 @@ def init_server(container):
 
 @app.get("/")
 def default():
-    return {"status": "OK"}
+    raise Exception
```

Comment:
This returns Generic Error while calling "localhost:8008/", shouldn't be changed to something like Not Found?

Reply:
This is a mistake I will come back to a default response. Would you see something better than {"status": "OK"} ?